sieves/tasks/predictive/ner/bridges.py

```diff
@@ -0,0 +1,524 @@
+    @property
+    def inference_mode(self) -> ollama_.InferenceMode:
+        return ollama_.InferenceMode.chat
+
+    @property
+    def prompt_signature(self) -> type[pydantic.BaseModel]:
+        return self._prompt_signature
+
+
+class LangChainNER(PydanticBasedNER[langchain_.InferenceMode]):
+    @property
+    def inference_mode(self) -> langchain_.InferenceMode:
+        return langchain_.InferenceMode.structured_output
+
+    @property
+    def prompt_signature(self) -> type[pydantic.BaseModel]:
+        return self._prompt_signature
+
+
+class InstructorNER(PydanticBasedNER[instructor_.InferenceMode]):
+    @property
+    def inference_mode(self) -> instructor_.InferenceMode:
+        return instructor_.InferenceMode.chat
+
+    @property
+    def prompt_signature(self) -> type[pydantic.BaseModel]:
+        return self._prompt_signature
+
+
+class GliXNER(NERBridge[list[str], glix_.Result, glix_.InferenceMode]):
+    def __init__(
+        self,
+        entities: list[str],
+        task_id: str,
+        prompt_template: str | None,
+        prompt_signature_desc: str | None,
+    ):
+        """
+        Initializes GliXNER bridge.
+        :param entities: List of entity types to extract.
+        :param task_id: Task ID.
+        :param prompt_template: Custom prompt template.
+        :param prompt_signature_desc: Custom prompt signature description.
+        """
+        super().__init__(
+            entities=entities,
+            task_id=task_id,
+            prompt_template=prompt_template,
+            prompt_signature_desc=prompt_signature_desc,
+        )
+
+    @property
+    def _prompt_template(self) -> str | None:
+        return None
+
+    @property
+    def _prompt_signature_description(self) -> str | None:
+        return None
+
+    @property
+    def prompt_signature(self) -> list[str]:
+        return self._entities
+
+    @property
+    def inference_mode(self) -> glix_.InferenceMode:
+        return glix_.InferenceMode.ner
+
+    def integrate(self, results: Iterable[glix_.Result], docs: Iterable[Doc]) -> Iterable[Doc]:
+        docs_list = list(docs)
+        results_list = list(results)
+
+        class Entity(pydantic.BaseModel):
+            text: str
+            start: int
+            end: int
+            entity_type: str
+
+        class Entities(pydantic.BaseModel):
+            text: str
+            entities: list[Entity] = []
+
+        for doc, result in zip(docs_list, results_list):
+            entities_list = []
+            # Get the original text from the document
+            doc_text = doc.text if doc.text is not None else ""
+
+            if result:
+                for entity in result:
+                    if isinstance(entity, dict) and "text" in entity and "label" in entity:
+                        # Convert types safely
+                        try:
+                            entity_text = str(entity["text"]) if entity["text"] is not None else ""
```

Comment:
- Why do we need to convert `entity["text"]` to string?
- More readable:
```suggestion
                            entity_text = str(entity["text"] or "")
```